app/styles/classes.less

```diff
@@ -30,6 +30,15 @@
   width: 100%;
 }
 
+// Margins
+.margin-bot--20 {
+  margin-bottom: 20px;
+}
+
+.margin-right--20 {
+  margin-right: 20px;
+}
```

Comment:
We should use a less loop to generate these helper classes for for all sides (top, bottom, left, right, vertical, horizontal) at some base unit of our choosing.

Reply:
This doesn't have to be part of this PR, was just a general comment.

Reply:
Thanks- will change for next time